tensorflow_serving/batching/batching_session.cc

```diff
@@ -102,6 +105,127 @@ TensorSignature TensorSignatureFromSignatureDefs(
   return tensor_signature;
 }
 
+template<int num_dims>
+Eigen::array<std::pair<int32, int32>, num_dims> CreatePadding(Tensor tensor,
```

Comment:
Would be better to use the Tensor abstraction, rather than the underlying eigen one, if that's possible.